packages/sources/peregrine-fund-admin/src/config/index.ts

```diff
@@ -0,0 +1,47 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
```

Comment:
Treat this as config file not code, we would just revert back to your previous implementation

Just keep API_NAV_ENDPOINT and API_RESERVE_ENDPOINT each default to a string and use them in the code

Reply:
Thank you, reduced to prior